providers/song-controls-front.js

```diff
@@ -0,0 +1,16 @@
+const { ipcRenderer } = require("electron");
+
+let videoStream = document.querySelector(".video-stream");
+module.exports = () => {
+    ipcRenderer.on("playPause", () => {
+        if (!videoStream) {
+            videoStream = document.querySelector(".video-stream");
+        }
+
+        if (videoStream.paused) {
+            videoStream.play();
+        } else {
+            videoStream.yns_pause();
```

Comment:
`Uncaught TypeError: videoStream.yns_pause is not a function` shouldn't it be
```suggestion
            videoStream.pause();
```
?

Reply:
Thats weird... where did you get that error?
`pause()` doesn't work on the youtube video stream. 
`yns_pause()` is a function that I extracted from the actual youtube video stream and it works in all my tests

Reply:
Maybe it depends on the platform 🤔  -> A workaround can be to check if `yns_pause` is an existing function, in which case it can be called, otherwise call `pause`.
Agreed about using it only if `auto-confirm-when-paused` isn't enabled, it's not perfect but this way it won't break stuff!

Reply:
If you write in dev console `document.querySelector(".video-stream").pause()` while music is playing, does it pause for you?

I'm asking because if `yns_pause()` doesn't exist and `pause()` doesn't work, then a user wouldn't be able to pause

Reply:
I Tested this with `auto-confirm-when-paused` and it seems to work flawlessly for me.

Is it maybe possible you didn't update your locally deployed version of `YoutubeNonStop`?
If not - I'm really confused as to how it can be that it works for me but not for you... platform difference again??